ppr-ui/src/components/tables/mhr/SearchedResults.vue

```diff
@@ -341,6 +341,10 @@ export default defineComponent({
         return result.includeLienInfo !== true ? { ...result, includeLienInfo: false } : result
       })
       localState.totalResultsLength = resp.totalResultsSize
+      if (localState.searchType === UISearchTypes.MHR_NUMBER && localState.totalResultsLength === 1) {
+        // Select search result if an MHR Number Search and search results equals 1.
+        localState.results = localState.results.map(result => ({ ...result, selected: true }))
```

Comment:
If there is only one result, do we even need to use map() ?

Reply:
I did this to keep with existing logic... It may work but I have not tested..